api/apps/v1alpha1/nimcache_types.go

```diff
@@ -72,19 +72,22 @@ type NIMSource struct {
 
 // +kubebuilder:validation:XValidation:rule="(has(self.modelName) ? 1 : 0) + (has(self.datasetName) ? 1 : 0) == 1",message="Exactly one of modelName or datasetName must be defined"
 type DSHFCommonFields struct {
-	// modelName is the name of the model
+	// ModelName is the name of the model
```

Comment:
Changes which are already in `main` are being shown as part of this PR. This generally happens because Github is not able to detect a common base correctly. Can you update your branch with main and rebase your changes on top of it.